### PR TITLE
User/ideen/tel 30

### PR DESCRIPTION
**Pull request by ideen1**

### CODE REVIEW

File: web-app/src/PacketContext.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    return (
+        <packetContext.Provider value={{ currentPacket, setCurrentPacket }}>
+            {children}
+        </packetContext.Provider>
+    )
+}
+
+export function usePacket() {
+    return useContext(packetContext);
+}
```
Comment (Dillonsd) [inline web-app/src/PacketContext.tsx:22]: Newline

File: web-app/src/objects/telemetry-data.interface.ts [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+      DischargeRelayEnabled: boolean,
+      ChargeRelayEnabled: boolean,
+      ChargerSafetyEnabled: boolean,
+      MalfunctionIndicatorActive: boolean,
+      MultiPurposeInputSignalStatus: boolean,
+      AlwaysOnSignalStatus: boolean,
+      IsReadySignalStatus: boolean,
+      IsChargingSignalStatus: boolean
+  }
+
```
Comment (Dillonsd) [inline web-app/src/objects/telemetry-data.interface.ts:259]: New line

**Review by Dillonsd — CHANGES_REQUESTED** [review]
Small styling.

**Review by JensVarughese — COMMENTED** [review]
It would be nice to have a configExample.ts, like in the server so people will know the format of Config.ts

**Review by jjason-nn — APPROVED** [review]
Looks good

**Review by Dillonsd — APPROVED** [review]

**Pull request merged**